src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.html

```diff
@@ -1,3 +1,10 @@
+<!-- loading spinner -->
```

Comment:
I think this is from another ticket

Reply:
You are right! I wanted to test st by adding it here.

Reply:
reverted.